PARCEL_2_RFC.md

```diff
@@ -0,0 +1,1285 @@
+# Parcel 2 RFC
+
+This RFC is written as documentation for the project, our documentation should
+be detailed enough to work as a spec, and it's easier to write it once this way
+than to approve a spec and start again on the documentation from scratch.
+
+Everything in Parcel should be documented here, if it is not documented it will
+not be part of Parcel 2.
+
+---
+
+## Introduction
+
+Parcel is a compiler for all your code, regardless of the language or toolchain.
```

Comment:
"all your code" sounds too broad, consider "all your web application frontend code".

Reply:
Parcel can also be used for back-end code, mainly node

Reply:
We'd like to keep our options open. :) Perhaps something like react native could be packaged with Parcel. The architecture has nothing JavaScript or even Web specific in it, so it should be possible to extend to any platform.